frontend/src/hooks/api/secrets/queries.tsx

```diff
@@ -131,7 +139,11 @@ export const useGetProjectSecretsAllEnv = ({
 
   const secrets = useQueries({
     queries: envs.map((environment) => ({
-      queryKey: secretKeys.getProjectSecret({ workspaceId, environment, secretPath }),
+      queryKey: secretKeys.getProjectSecretAllEnvironments({
+        workspaceId,
+        environment,
+        secretPath
+      }),
```

Comment:
Since we are no longer using `select` we need a separate query key for fetching all environments as the return data is now different - still invalidates from the original key but has a separate cache.